qiskit_nature/algorithms/ground_state_solvers/minimum_eigensolver_factories/vqe_ucc_factory.py

```diff
@@ -33,23 +38,48 @@
 from ...initial_points import InitialPoint, HFInitialPoint
 from .minimum_eigensolver_factory import MinimumEigensolverFactory
 
+logger = logging.getLogger(__name__)
 
-class VQEUCCFactory(MinimumEigensolverFactory):
-    """A factory to construct a VQE minimum eigensolver with UCCSD ansatz wavefunction."""
 
+class VQEUCCFactory(MinimumEigensolverFactory):
+    """A factory to construct a VQE minimum eigensolver with UCCSD ansatz wavefunction.
+
+    Note: get_solver will overwrite any value that we directly set onto the vqe for both
+    the ansatz and the initial point. For example:
+    .. code-block:: python
+
+        factory = VQEUCCFactory()
+        factory.minimum_eigensolver.ansatz = UCCD()
+        vqe = factory.get_solver()
+        print(type(vqe.ansatz))  # UCCSD
```

Comment:
And again, add something like the following:
```python
factory.ansatz = UCC(excitations="d")
```

Reply:
Well if you were to write `UCCSD()` there, the example would not show any obvious difference. I simply wanted to correct the `UCCD` example :+1: